XSC/be/CXX/Header.cpp

```diff
@@ -943,6 +945,15 @@ generate_header (Context& ctx,
   Includes includes (ctx);
   Traversal::Names schema_names;
 
+  if (!ctx.cpp11())
+  {
+    ctx.os << "#if defined(ACE_HAS_CPP11)" << endl
+           << "# define XML_XSC_SMART_PTR(X) std::unique_ptr<X>" << endl
```

Comment:
Shouldn't it check that XML_XSC_SMART_PTR is not defined yet in case user code includes multiple headers that all define XML_XSC_SMART_PTR?

Reply:
I'll add that.